lib/project_types/script/tasks/ensure_env.rb

```diff
@@ -0,0 +1,77 @@
+require "shopify_cli"
+
+module Script
+  module Tasks
+    class EnsureEnv < ShopifyCli::Task
+      attr_accessor :ctx
+
+      def call(ctx)
+        self.ctx = ctx
+
+        script_project_repo = Layers::Infrastructure::ScriptProjectRepository.new(ctx: ctx)
+        script_project = script_project_repo.get
+
+        return if script_project.api_key && script_project.api_secret && script_project.uuid
```

Comment:
If the `api_key` is set, should we still prompt for the app? Or should we just use that app key and prompt for the script?

Reply:
I chose to prompt for the app just to keep things simple. We could use the app key that’s already provided, but it makes the logic less elegant since the key and the secret are related, and now we introduce different side effects (if api_key and secret are present, prompt only for UUID, but if only UUID is present, then prompt for api_key, secret, and also UUID because the previous UUID would be invalid). 

The CLI also ensures that the .env is created correctly, so the cases where we would only prompt the user for the UUID would be when using a Script created with an older version of the CLI, or if the developer changes the .env manually. So I’m not sure if its worth the effort